### DISCUSSION TITLE
`prevent-element-src-loading` scriptlet support

### DISCUSSION THREAD

**Discussion by superlex** [uBlockOrigin/uBlock-issues#3026] [Interoperability] [answered]
Hello,
does uBlock Origin support `prevent-element-src-loading`scriptlet?
If it doesn't, can I ask to support it?

AdGuard scriptlet:
https://github.com/AdguardTeam/Scriptlets/blob/master/wiki/about-scriptlets.md#-%EF%B8%8F-prevent-element-src-loading

Description:
Prevents target element source loading without triggering 'onerror' listeners and not breaking 'onload' ones.

Example where it can be used:
https://www.capital.it/programmi/extra/puntate/extra-del-30-05-2023/
AG solution:
`capital.it#%#//scriptlet('prevent-element-src-loading', 'script', 'static.chartbeat.com/js/chartbeat_brightcove_plugin.js')`
uBO solution:
`@@||static.chartbeat.com/js/chartbeat_brightcove_plugin.js$domain=capital.it`

Consider that `chartbeat.com` is blocked by Peter Lowe list and EasyPrivacy, and it is unlocked by this last one.

**Comment by gorhill** [comment, marked as answer]
Redirecting to a noop resource doesn't work?

**Comment by superlex** [comment]
It works, thanks!
`||static.chartbeat.com/js/chartbeat_brightcove_plugin.js$script,redirect-rule=noopjs,domain=capital.it`
